packages/grafana-e2e/cypress/support/commands.ts

```diff
@@ -31,3 +31,11 @@ Cypress.Commands.add('getJSONFilesFromDir', (dirPath: string) => {
     relativePath: dirPath,
   });
 });
+
+Cypress.Commands.add('startBenchmarking', (testName: string) => {
```

Comment:
Should these command be added if  BENCHMARK_PLUGIN_ENABLED is false? Or maybe we at least should warn the user if that is the case?

Reply:
I think the methods should be there since they are present in the namespace declaration file (https://github.com/grafana/grafana/blob/31e97cb15063a9814fe0a695935980f37e3b48ce/packages/grafana-e2e/cypress/support/index.d.ts#L9-L10) but it makes sense to add warning logs - will add